modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/servicetask/JavaServiceTaskTest.java

```diff
@@ -132,36 +135,29 @@ public void testExpressionFieldInjectionWithSkipExpression() {
         ProcessInstance pi2 = runtimeService.startProcessInstanceByKey("expressionFieldInjectionWithSkipExpression", vars2);
         Execution execution2 = runtimeService.createExecutionQuery().processInstanceId(pi2.getId()).activityId("waitState").singleResult();
 
-        assertNotNull(execution2);
+        assertThat(execution2).isNotNull();
 
         Map<String, Object> pi2VarMap = runtimeService.getVariables(pi2.getProcessInstanceId());
-        assertFalse(pi2VarMap.containsKey("var1"));
-        assertFalse(pi2VarMap.containsKey("var2"));
+        assertThat(pi2VarMap)
+                .doesNotContainKey("var1")
```

Comment:
Shouldn't this be `containsKey`?

Reply:
@filiphr No, note that the original assertion was `assertFalse`.

Reply:
Yes you are 100% right. I only read the `containsKey`. My mistake sorry.